compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/util/DeclarationStubGenerator.kt

```diff
@@ -76,8 +76,8 @@ class DeclarationStubGenerator(
         if (symbol is IrFieldSymbol && (symbol.descriptor as? WrappedPropertyDescriptor)?.isBound() == true) {
             return generateStubBySymbol(symbol, symbol.descriptor)
         }
-        val descriptor = if (symbol.descriptor is WrappedDeclarationDescriptor<*>)
-            findDescriptorBySignature(symbol.signature)
+        val descriptor = if (symbol.descriptor is WrappedDeclarationDescriptor<*> && symbol.signature != null)
```

Comment:
Why an additional condition is required here?

Reply:
Does WrappedDeclarationDescriptor guarantee that symbol has signature?

Reply:
I understood that before it was supposed to be so. But I don't see guarantees in code. May be made this condition as require or assert?

Reply:
No there is no such guarantee in general. It could be symbol for private declaration.
I think it worth to add something like 
` symbol.signature ?: error("expected signature... bla bla bla")`